soh/src/overlays/actors/ovl_En_Insect/z_en_insect.c

```diff
@@ -211,6 +211,13 @@ void EnInsect_Init(Actor* thisx, PlayState* play2) {
 
         func_80A7D39C(this);
 
+        // For bugs that aren't linked to a soil patch, we remove the "short lived" flag to prevent them from despawning
+        // And exit early to not increment the "bugs dropped count"
+        if (CVarGetInteger("gNoBugsDespawn", 0) && this->soilActor == NULL) {
+            this->unk_314 &= ~4;
```

Comment:
would love to see this not be magic numbers but considering the `unk` i figure that can come in with a docs pull at some point

Reply:
Yeah this entire actor file is wonderfully doc'ed in decomp upstream 😉 (I referenced decomp for these changes)
Updating against decomp should be a nice freebie for the develop branch later.
(I have a short list of a few files that I'd like to update once we do another macready->dev)